### ISSUE TITLE
[Bug]: [Flatpack] Downloading caffe fails to download repeatedly on inital install

### ISSUE THREAD

**Issue by MaximumMaxxx** [bottlesdevs/Bottles#1345]
### Describe the bug

After installing bottles and trying to do the initial runner download it gets stuck in a loop.

Error:
```
...
(20:08:21) WARNING No runners found. 
(20:08:22) INFO Installing component: [caffe-7.6]. 
Downloading caffe-7.6-x86_64.tar.xz: 100% [==================================================>

(20:08:29) INFO Renaming [caffe-7.6-x86_64.tar.xz] to [caffe-7.6-x86_64.tar.xz]. 
(20:08:30) ERROR Downloaded file [caffe-7.6-x86_64.tar.xz] looks corrupted. 
(20:08:30) ERROR Source cksum: [9050f43bcb320fccba3b1dd9a25b9ac8] downloaded: [bcbcc047e908f75af1c8032ae492d487] 
(20:08:30) INFO Removing corrupted file [caffe-7.6-x86_64.tar.xz]. 
(20:08:30) ERROR Extraction failed! Archive ends earlier than expected. 
(20:08:30) INFO Runners found:
         - sys-wine-7.0
...
```
I keeps looping like this until I stop bottles

### To Reproduce

1. Install the flatpack
2. Run the inital setup to download the runners

### Package

Flatpak from Flathub

### Distribution

Kde Neon

### Health Check

```shell
Doesn't get far enough to allow for this
```

### Display Protocol

X11

### Display Backend used by Bottles

I don't know

**Comment by tnkmon** [comment]
The same error in pure archlinux (flatpak install)

**Comment by rjousse18** [comment]
Same here ! (Flatpak, Ubuntu 20.04)

**Comment by mirkobrombin** [comment]
Fixed, hash for caffe 7.6 was wrong. Also, 7.5 and 7.6 are now marked as unstable, so Bottles will download 7.4. Have a good day.
